.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+# command to install dependencies
+install: 
+  pip install -r pip-requirements.txt
+
+# command to run tests
+script: 
+  nosetests --with-coverage --cover-package=presenters,plotting.FigureManager
```

Comment:
Is this actually running your unit tests? I'm not convinced it is? All unit tests must be run as part of the CI. We need to get the outputs of the executed tests in the output console.